R/utils-cache.R

```diff
@@ -0,0 +1,57 @@
+#' Standardize text for hashing
+#'
+#' Make sure text after styling results in the same hash as text before styling
+#' if it is indeed identical.
+#' @param text A character vector.
+#' @keywords internal
+hash_standardize <- function(text) {
+  text <- ensure_last_is_empty(text)
+  Encoding(text) <- "UTF-8"
```

Comment:
Do you really want to override encoding?

```suggestion
  text <- enc2utf8(text)
```

Reply:
Right... Can't remember why I did it that way. At least it does not seem to break tests when accepting your solution so let's do it.